Zendesk Triage SOP

kozbo opened the pull request:
RFC describing how we will handle zendesk support tickets

kozbo committed:
Merge pull request #2 from HumanCellAtlas/master

update my base

kozbo committed:
initial version zendesk triage SOP

kozbo committed:
cleanup formatting

kozbo committed:
more cleanup

kozbo committed:
diagram

kozbo committed:
add diagram

kozbo committed:
wrong location

kozbo committed:
add image ref

kozbo committed:
wrong location

kozbo committed:
initial version

kozbo committed:
wrong name

kozbo committed:
renamed file

kozbo committed:
test image

kozbo committed:
image correction

tjchen commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,153 @@
+This section describes the flow of an issue through the support state
+diagram presented below.
+
+![](../images/0000-Zendesk-triage-SOP-diagram.jpeg)
```
This is great, would it be possible to add to the secondary support step some color on "Export to Github"? Sometimes it's not 1:1, sometimes it links to an existing feature request with a subsequent record, sometimes it becomes a bug, or an issue request, or something else. Happy to help give an example as well of how we can prioritize (P1/P2/P3).

malloryfreeberg replied:
I think an image here is a great idea. I have a few questions about the actual flow, though.

1. "User Issue Status: New" should technically not have any arrows coming into it, as it is the single input to the diagram. An issue is only "new" once; after someone responds to it at all, it is now "open". 
1. There are some answers missing from some of the arrows coming out of the diamond entities. 
1. There is a hanging arrow "Give solution to user" at the bottom of the diagram that needs to point to "closed" or something equivalent.
1. For some rectangle entities (not sure what these entities represent), there are multiple arrows coming out but it's unclear which arrow should be taken.

I'm happy to work on editing the diagram if someone could share an editable file with me :)

jodihir replied:
@malloryfreeberg Are you able to edit this doc?: https://www.lucidchart.com/documents/edit/f7cc3f5e-c306-40bc-8063-6aa3fb75e179/0?shared=true&
It's an updated version from Kevin- updated based on your list.  Feel free to edit more!

malloryfreeberg replied:
I can view the doc. Requested access to edit.

Changes look good!

lauraclarke commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,153 @@
+
+## Summary
+
+This process RFC describes how the DCP handles technical support requests through the use of the Zendesk application and other processes.
```
Why technical support requests, surely we will get scientific requests too and having two different policies feels cumbersome?

malloryfreeberg replied:
I agree with @lauraclarke . Some of the questions we've received so far have been scientific questions. For example:

> Can I request for more information on Melanoma infiltration data set that is made public. I request additional information on the causative agent/drug used to induce melanoma and age of the time point specific mouse might also be helpful. 

> I was wondering whether the immune census has associated clustering/celltyping data, that isn't present in the https://preview.data.humancellatlas.org/ page. e.g. a vector containing the cell types (imputed from the gene expression patterns) for each cell ID?

Maybe being clear here that this RFC proposal will handle scientific questions, as well, would be helpful.

matthewspeir replied:
Hmm, I can't remember if this is part of the SOP, but I think that we need to be clear about types of 'scientific questions' we will answer and what types we won't. I don't think that we should be answering questions that stray into scientific advice (e.g. How should I analyze the data I've downloaded? What clustering method is best? etc). And for the questions @malloryfreeberg listed, I think we should just be pointing these people to the submitters rather than attempting to answer these ourselves.

lauraclarke replied:
@matthewspeir you are correct that we won't be able to answer all scientific questions and may frequently need to direct people to the contributing lab or a domain expert but that is different from saying we will answer no scientific questions. We should be able to give people to approach with basic questions about the HCA and the data in the DCP and then point them to better resources for more detailed information and we should be able to answer questions about the data we provide and how it was analysed/annotated

jodihir replied:
I agree - I think this proposal was meant to cover the entirety of questions that come in through our data-help@humancellatlas.org email address (the address that feeds into the Zendesk ticketing system). Those can be technical or scientific. Maybe we should exchange the word "technical" here for "user", so "technical support" is just "user support"?

malloryfreeberg replied:
+1 for "user support".

As stated, while we may not be able to give answers to all scientific questions, we should have some kind of plan for responses to those types of questions. We don't need to figure out the response now (could be pointing to another resource or the data contributors), but we need to be clear that we will handle these requests in some form.

lauraclarke commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,153 @@
+
+## Motivation
+
+Beta 2 testing will begin soon and we’ve decided to collect feedback
```
This feels like a very short term motivation. Surely we need to define a policy which will work beyond beta2 and review and refine it as demands change rather than defining a very short term policy

jodihir replied:
Good point - I'll revise the wording of the motivation as mallory suggested below. I think this same plan can take us into GA.

lauraclarke replied:
Great, my feeling is that we should define a plan without any reference to specific timepoints but with a comment to review if the volume or type of queries require us to reassess the policy

kozbo replied:
I have re-worded this section a little further to remove the language about the Beta program.

lauraclarke commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,153 @@
+repository if the reporter is sure of which one to use. Once an issue is
+created in Zendesk there is a tiered support network that kicks in to
+resolve the question.
+
```
It would be good to make a decision between DCP staff members using zendesk or filing directly in an appropriate repo. This sort of if/else policy is likely to create confusion

jodihir replied:
This might be worth some discussion.  If DCP staff members file issues via Zendesk, that helps the triage staff know that an issue is known about and being addressed. (That can be useful if users write about the same issue.) On the other hand, the workload for triage staff increases, and they need to do something with that ticket. Thoughts?

malloryfreeberg replied:
> If DCP staff members file issues via Zendesk, that helps the triage staff know that an issue is known about and being addressed.

This is a good point. I think this would fall under the triage staff person "searching the knowledge base" for the solution. One of the places the person should search is in GitHub issues, seeing if a similar issue was already reported and/or solved. 

If we are already understaffed for ZenDesk, then you're right @jodihir we don't want to overburden with tickets from our own DCP staff. My opinion is that DCP staff members should not file ZenDesk tickets, but rather file them in GH.

jodihir replied:
Edited

lauraclarke commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,153 @@
+support requests from end users, and monitors the Zendesk system daily
+for new tickets. The triager takes ownership of each open ticket, sends
+out a canned response noting that we’ve received the user’s message, and
+looks to see if there is enough information in the ticket to inform the
```
Zendesk already sends out a response like this. We need to make sure if a triager also responds it contains something useful or different to this otherwise will be repetitive and annoying

"Your request (80) has been received and is being reviewed by our support staff.

To add additional comments, please reply to this email."

jodihir replied:
l edited the paragraph to address this.

lauraclarke commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,153 @@
+
+### **Acceptance Criteria \[optional\]**
+
+*Acceptance criteria are the conditions that a RFC must satisfy to be accepted by users or other stakeholders.*
```
Surely this needs some text?

jodihir replied:
I'll discuss with @kozbo

kozbo replied:
i have removed the optional acceptance criteria section

lauraclarke reviewed (changes requested):
Happy to chat about my comments in a phone call if that is helpful

malloryfreeberg commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,153 @@
+we have received to date. Several of us have been thinking about this
+for awhile and drew up an initial SOP that we refined at the last DCP
+F2F (Feb. 2019); we present it here as this RFC. 
+
```
Maybe we should break the Motivation section up into 2 paragraphs. One that highlights the immediate need (i.e. Beta2) and one that highlights the longterm sustainability need (i.e. handling all user queries post-Beta2).

jodihir replied:
I've expanded the Motivation section.

malloryfreeberg commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,153 @@
+not, then the triager needs to ask for help from the secondary support
+team, which includes members from each DCP component or service. It may
+be obvious which team should receive the ticket; if so, then it can be
+assigned directly. If it’s not obvious, there can be some internal
```
> If it’s not obvious, there can be some internal discussion with the secondary support staff about who should receive the ticket.

I worry that having "internal discussion" about this decision will lead to repeating the same discussions over and over (and potentially annoy people and waste their time). Can we commit to capturing these discussions somewhere transparent so that when future tickets come in regarding the same type of topic, the triage person can know for certain which team the ticket should be assigned to?

We don't necessarily need to define what the transparent mechanism is now in this SOP, but it would be good to be clear that we plan to have these discussions recorded somewhere

jodihir replied:
I've changed this section to enumerate the specific points, including this one.

malloryfreeberg commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,153 @@
+new/updated FAQs, code examples, bug reports, etc.
+
+The triager(s) will also work with the secondary support team to create
+and maintain canned responses (how to connect/get involved with the HCA;
```
> maintain canned responses

Is the idea to maintain these as part of a FAQ section? It would be great if this was not an internal record of canned responses but rather public-facing documentation so that the next person that asks the same question can be pointed to the documentation rather than the triage person copy/pasting from another source.

jodihir replied:
Yes, I agree. I added the following sentence to the points now listed in the doc:
As much as possible, issues should be turned into documentation or short FAQs and made available for users to access from the portal.

jodihir committed:
Update 0000-zendesk-triage-sop.md

test heading

jodihir committed:
Update 0000-zendesk-triage-sop.md

expanded the Motivation section

jodihir committed:
Update 0000-zendesk-triage-sop.md

edited to include a plan for what to do with internal tickets

jodihir committed:
Update 0000-zendesk-triage-sop.md

edits made based on comments

jodihir committed:
Update 0000-zendesk-triage-sop.md

small edits

jodihir committed:
Update 0000-zendesk-triage-sop.md

changed remaining instances of "Technical Support" to "User Support"

kozbo committed:
Minor edits

Removed language about the Beta program and removed the Acceptance criteria section.

kozbo commented:
@lauraclarke I think I have resolved all the comments now. Will you accept it now? I can't seem to resolve it.

kozbo committed:
Update optional sections at the end

matthewspeir reviewed (approved):
Everything looks good to me.

kozbo commented:
@lauraclarke bump, I think I have addressed your comments. Are you willing to close out your review now?

malloryfreeberg commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+> knowledge base. They would also be responsible for moderate our user
+> forum.
+> 
+> 1 point person per box (4-5 people) - this should be at least 25% of
```
I'm generally happy with the SOP - the updates seem very reasonable! I wanted to mention that the "at least 25% of their time" strikes me as being a bit high, and probably will depend on the component? If I were the point person (I think this is the Secondary Support team member?) for metadata or wrangling, this would be 2 hours out of my day each day, which seems a bit over kill :) I understand the desire to put something concrete here, so again, happy to merge this SOP as-is and maybe think a bit about the time commitment.

lauraclarke replied:
I agree that with the current level of tickets (1 a week) 25% of 5 peoples time seems like a lot. That is 1.25FTEs committed to this on top of the 1FTE dedicated personnel

I would suggest not putting a time commitment on it with the current level of tickets but noting the PO/PL who owns zendesk to monitor and ask for a defined commitment once the ticket number gets close to 1-2 a day

lauraclarke commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+## Motivation
+
+### Immediate Need 
+Users must have a stadard way to guve feedback and request help. There will be a general up-tick on support demand every time we release a new feature or roll out a new type of data. Having a basic ticketing system in place will help assure that no problems are missed. Understanding the needs of the DCP users is paramount. 
```
Typos to fix

brianraymor replied:
Specifically, _stadard_ and _guve_.

malloryfreeberg replied:
Fixed.

lauraclarke reviewed (approved):
A couple of small comments have been made. I am happy to see this move forward assuming they are addressed

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+Users must have a stadard way to guve feedback and request help. There will be a general up-tick on support demand every time we release a new feature or roll out a new type of data. Having a basic ticketing system in place will help assure that no problems are missed. Understanding the needs of the DCP users is paramount. 
+
+### Sustainability Through the GA
+Upon general release, the full range of use cases will come into play and we anticipate a large increase in both the nubmer of tickets and diversity of questions asked by users. Addressing these tickets will require a big time committment, so we need an efficient system in place and a team well-versed in the DCP and its scientific applications. 
```
How is _general release_ defined?

I would recommend replacing _we_ with the DCP or avoiding its use. For example, _A large increase in both the number of tickets and diversity of questions is anticipated_. 

_nubmer_->_number_
_committment_->_commitment_

malloryfreeberg replied:
Fixed typos.

malloryfreeberg replied:
> I would recommend replacing we with the DCP or avoiding its use.

Updated.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+### Sustainability Through the GA
+Upon general release, the full range of use cases will come into play and we anticipate a large increase in both the nubmer of tickets and diversity of questions asked by users. Addressing these tickets will require a big time committment, so we need an efficient system in place and a team well-versed in the DCP and its scientific applications. 
+
+This RFC describes an SOP that we think will satisfy both the Beta2 test phase and the early days of the GA. As we grow in our understanding of user support needs, this SOP will likely need revision to address long-term sustainability.
```
I think that the RFC should avoid referencing _GA_.

kozbo replied:
GA has taken on many meanings to many people, but ultimately it means when the prod.data.humancellatlas.org becomes data.humancellatlas.org. We will be redirecting the Preview site to that address at the same time. This means that the DCP will be generally available. I will add some text to clarify what I mean by GA to the PR.  Let me know if you still object. (making several changes at once so give me some time.)

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+
+Issues in Zendesk originate from users of the DCP. These users could
+include data contributors, researchers trying to use the data in the
+DCP, software developers trying to leverage the DCP API. Any user of the
```
, and _software developers ..._

malloryfreeberg replied:
Fixed.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+
+## Issue Creation
+
+Issues in Zendesk originate from users of the DCP. These users could
```
Suggest _Any user of the DCP can file an issue in ZenDesk. These users include ..._ Otherwise, the opening sentence and closing sentence in the paragraph are a redundant.

malloryfreeberg replied:
Fixed.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+The triager takes ownership of each open ticket and does the following to develop a response:
+
+1. Evaluate whether there is enough information in the ticket to inform the
+response. If not, write back to the user and ask for the missing information. 
```
How long do you wait for a response before closing the issue? Also - do you want to use Zendesk lingo - agent instead of triager?

kozbo replied:
Added 2 weeks for the user to respond. I prefer to stay with terms that are not specific to Zendesk as we might change tools one day.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+1. Evaluate whether there is enough information in the ticket to inform the
+response. If not, write back to the user and ask for the missing information. 
+
+2. Search the data portal documentation and FAQ for answers to the question. If the triage person can find an answer that resolves the problem, a repsonse is sent to the user and the issue is closed. 
```
triager vs triage person?

_repsonse_->_response_

malloryfreeberg replied:
Fixed.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+
+### **Unresolved Questions**
+
+We suggest that funding be sought to hire one or two part or full-time
```
_GA_ ...

malloryfreeberg replied:
Updated.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+
+## Primary Support
+
+The triage person, or triage team, is responsible for receiving initial
```
Without funding, who is primary support?

kozbo replied:
So far it has been Mallory and I :-| 
But you are correct. I will say that that this position must be funded.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+
+### **Alternatives**
+
+No other alternatives have been proposed.
```
One alternative would be to *not* have a support desk unless it's funded. As I noted in Jodi's earlier proposals, this type of frontline-backline support can substantially impact development teams.

kozbo replied:
saying "that the alternative is no support desk, and no one person or team responsible for working through user needs" seems so patently unworkable that I don't want to add it into the RFC.  It is a bit of a challenge in that the RFC can be accepted but not funded, so that won't work either.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+3. If an answer is not evident from searching the knowledge base in the data portal, the triager needs to ask for help from the secondary support team, which includes members from each DCP component or service. It may
+be obvious which team should receive the ticket; if so, then it can be
+assigned directly. If it’s not obvious, there can be some internal
+discussion with the secondary support staff about who should receive the
```
Bit _hand wavy_ - for example, there's no mention of the zendesk slack channel? What happens if secondary support ignores the request because they're busy shipping code?

kozbo replied:
I took a shot at resolving this.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+discussion with the secondary support staff about who should receive the
+ticket.
+
+4. At this point the ticket becomes the responsibility of the secondary support person, who should repond to the user and bring the ticket to an eventual close. 
```
_repond_->_respond_ Why should and not MUST? What happens if they don't respond?

kozbo replied:
updated to add the SLA and "MUST"

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+
+
+In order to improve our User Support the triager will also work with the secondary support team to:
+
```
For the section below, I'm unclear who is performing the actions based on _the triager will ... work with ... team to_ - how will they work with the team?

kozbo replied:
I think I fixed this section

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+The secondary support team is comprised of 1-2 members from each major
+component or service of the DCP. These representatives should have
+enough technical knowledge to answer most questions. Answers generated
+by the secondary support team will be added to the knowledge base. They
```
_added to the knowledge base_ by whom?

kozbo replied:
by the 2ndary team under the guidance of the triager.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+
+2. Search the data portal documentation and FAQ for answers to the question. If the triage person can find an answer that resolves the problem, a repsonse is sent to the user and the issue is closed. 
+
+3. If an answer is not evident from searching the knowledge base in the data portal, the triager needs to ask for help from the secondary support team, which includes members from each DCP component or service. It may
```
What is the _knowledge base_? Is this referring to documentation in the portal?

kozbo replied:
added a definition section.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+2. create and maintain canned responses (how to connect/get involved with the HCA;
+how to submit data, etc.) that can be quickly sent to users who have written with those questions.
+
+3. document discussions and decisions that arise from particular tickets, so that it's clear who should respond to similar questions in the future and what the response should be.
```
Document where?

kozbo replied:
added knowledge base

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,150 @@
+
+1. tag each ticket with basic tags (data store/ingest/contribution/etc.) that help DCP staff categorize it for later retrieval. In addition, Github tickets should be made for issues like: updating docs or writing new documentation and improving our FAQs, code examples, bug reports, etc. As much as possible, issues should be turned into documentation or short FAQs and made available for users to a ...
+
+2. create and maintain canned responses (how to connect/get involved with the HCA;
```
Create and maintain where?

kozbo replied:
added knowledge base

malloryfreeberg committed:
Fixed typo.

malloryfreeberg committed:
Fixed typos.

malloryfreeberg committed:
Fixed typos.

malloryfreeberg committed:
Fixed typos.

malloryfreeberg committed:
Fixed typos. Updated text.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,138 @@
+
+This process RFC describes how the DCP handles user support requests through the use of the Zendesk application and other processes.
+
+## Author(s)
```
The ` needs to be removed from the addresses below.

kozbo replied:
done

kozbo committed:
Address comments from Brian and Laura

lots of changes!

kozbo commented:
@brianraymor are you satisfied with my changes and ready to mark this reviewed? Ready for oversight?

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+
+[Mallory Freeberg](mailto:mfreeberg@ebi.ac.uk)
+
+## Shepherd
```
If you're the shepherd, please add your name here.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+    > a timely manner or be assured that someone else is responding in a
+    > timely manner.
+
+## Scientific "guardrails" \[optional\]
```
If there are no _guardrails_, then remove this section.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+## Scientific "guardrails" \[optional\]
+
+## Definitions
+`Knowledge Base` - a document that tracks all questions and answers that have been generated through the support infrastructure. This document can be mined to generate FAQs, or finding answers to obscure problems that have been solved in the past.
```
If the RFC is intended to be self-contained documentation for the zenhub process, then the location (google doc, github, etc) needs to be defined.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+This section describes the flow of an issue through the support state
+diagram presented below.
+
+![](../images/0000-Zendesk-triage-SOP-diagram.jpeg)
```
Remember to update the 0000 prefix to the assigned RFC number so that the image can be related to its RFC. Just opened #84 to clarify that point.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+
+## Issue Creation
+
+Any user of the DCP can file an issue. Users can include data contributors, researchers trying to use the data in the DCP, and software developers trying to leverage the DCP API. 
```
Perhaps _external user_?

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+
+Any user of the DCP can file an issue. Users can include data contributors, researchers trying to use the data in the DCP, and software developers trying to leverage the DCP API. 
+
+While problems identified by an internal DCP staff member could be filed on Zendesk too, we favor a plan in which those issues are directly added to the applicable development repository, where they can be addressed directly. This plan eliminates the extra work for the triager of managing internally-generated tickets.
```
I think this should be more strict. Internal DCP staff MUST NOT file Zendesk issues. The guidance should be crisp as possible. 

_Problems identified by DCP community members MUST not be submitted through the Zendesk process, but filed in the appropriate github repository, where they can be directly addressed by DCP teams._

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+
+2. Search the data portal documentation and FAQ for answers to the question. If the triage person can find an answer that resolves the problem, a response is sent to the user and the issue is closed. 
+
+3. If an answer is not evident from searching the Knowledge Base in the data portal, the triage person needs to ask for help from the secondary support team. The secondary support team is comprised of at least one member from each DCP component or service. Communication with the team is facilitated via the `zendesk` slack channel on [humancellatlas.slack.com](humancellatlas.slack.com). It may be  ...
```
The document is part of the data portal?

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+
+3. If an answer is not evident from searching the Knowledge Base in the data portal, the triage person needs to ask for help from the secondary support team. The secondary support team is comprised of at least one member from each DCP component or service. Communication with the team is facilitated via the `zendesk` slack channel on [humancellatlas.slack.com](humancellatlas.slack.com). It may be  ...
+
+4. At this point the ticket becomes the responsibility of the secondary support person, who should respond to the user and bring the ticket to an eventual close. The secondary support person MUST respond to the request within 2 days. 
```
_MUST respond_ -or- what happens? _SHOULD_ ?

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+
+## Secondary Support
+
+The secondary support team is comprised of 1-2 members from each major component or service of the DCP. These representatives should have enough technical knowledge to answer most questions. They may require additional information from the person who created the ticket. They also may need to consult with technical experts who make up the tertiary support teams. Answers generated by the secondary  ...
```
Are you trying to say that Secondary is Component PM and tertiary is Component TL? Could we simply eliminate tertiary? What doesn't the triager simply update the KB based on the response from secondary?

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+## Secondary Support
+
+The secondary support team is comprised of 1-2 members from each major component or service of the DCP. These representatives should have enough technical knowledge to answer most questions. They may require additional information from the person who created the ticket. They also may need to consult with technical experts who make up the tertiary support teams. Answers generated by the secondary  ...
+
```
I keep running into cases where an expression is used like _Tertiary Support_ that has a forward reference which makes reading difficult.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+
+## Tertiary Support
+
+The Tertiary team is the entire DCP team. In the event that the secondary support team is not able to respond to the question, anyone may be called on to answer it. However, the need for Tertiary support should be rare, and all answers will be archived and added to the Knowledge Base
```
The _entire DCP team_ is different from _technical experts_.

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+### **Funding**
+Support can have a significant impact on development teams. Without funding development velocity will be much slower. Also using trained support staff will often result in a much better user experience.
+
+We suggest that funding be sought to hire one full time or two part-time triage person(s) who would grow into the position. They should be able to handle primary triage early on as well as answer some of the more difficult questions, especially as the need for support increases over time. We will need a backup person for when the support person is on leave.
```
Who is making the suggestion and to whom?

brianraymor commented on rfcs/text/0000-zendesk-triage-sop.md:
```diff
@@ -0,0 +1,133 @@
+
+> 2 triage agents - could be up to 50% of their job as they would trade off
+> being point for this job. Duties would include the grooming of the FAQ
+> Knowledge Base. They would also be responsible for moderate our user
```
_moderate_ -> _moderating_ - what is our _user forum_?

brianraymor reviewed (changes requested):
I don't believe that the RFC offers enough clarity over the current ad-hoc process. For example, there's a lack of clear role assignments and basic details. Someone may volunteer to be the triager today, but not tomorrow. (Consider the transitions in the author's list for example)

It may be helpful to review the support question with _Oversight_ to ensure that the institutions are signed up for their development teams taking on a support role and to discover if funding is available for a primary support team. I provided similar feedback to Jodi's communication team plan. The approval needs to occur at a higher level.

kozbo committed:
original rtf format

kozbo committed:
first md format file

kozbo committed:
removing file

kozbo committed:
removing file

kozbo committed:
converted md file

kozbo committed:
basic cleanup from conversion

kozbo commented:
This RFC has been rejected. The implementation requires that we put a full time person (or 2 people) in place, and there isn't the need for that level of staffing yet. The data operations team has taken on the role of triaging issues. 
Recommend that we rewrite with the process described, but scale back to current staff.
